oe_multilingual.module

```diff
@@ -212,3 +212,12 @@ function oe_multilingual_page_header_pre_render(array $build): array {
 function oe_multilingual_locale_translation_projects_alter(&$projects) {
   $projects['oe_multilingual']['info']['interface translation server pattern'] = drupal_get_path('module', 'oe_multilingual') . '/translations/%project-%language.po';
 }
+
+/**
+ * Implements hook_language_negotiation_info_alter().
+ */
+function eyp_language_negotiation_info_alter(&$negotiation_info) {
```

Comment:
Function prefix is not correct: eyp -> oe_multilingual

Reply:
Fixed.